### ISSUE TITLE
newsmaxtv.com: breakage

### ISSUE THREAD

**Issue by NewUser12345123** [uBlockOrigin/uAssets#18009]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.newsmaxtv.com/`

### Category

breakage

### Description

Everything was working fine forever until today.
All of the sudden, Newsmaxtv.com has no audio on firefox.

When uBO is enabled, you are unable to hear audio even with audio enabled.

It only occurs on firefox
On Chrome, uBO and Newsmaxtv.com work fine
The newsmaxtv youtube stream is also fine

when i turn off uBO and refresh page, audio still doesnt work on Firefox+newsmax
I have to close firefox, then open again with uBO off on newsmax, in order to hear the audio and then it works

turn on uBO and refresh page and no audio again

So it happened yesterday/today that it just stopped working
Its only happening on firefox
And im only experiencing the issue on Newsmaxtv.com

also i have no filters enabled other than just uBO standard

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.49.2
Firefox: 112
filterset (summary): 
 network: 93350
 cosmetic: 40781
 scriptlet: 17387
 html: 947
listset (total-discarded, last-updated): 
 default: 
  user-filters: 361-39, never
  ublock-filters: 33946-52, 2h.40m
  ublock-badware: 7111-0, 39m
  ublock-privacy: 443-0, 7d.3h.43m
  ublock-abuse: 70-70, 7d.5h.45m
  ublock-unbreak: 2021-2, 2d.4h.51m
  ublock-quick-fixes: 247-0, 10h.41m
  easylist: 67082-631, 20h.3m
  easyprivacy: 32236-609, 5d.4h.7m
  urlhaus-1: 6768-1, 16h.15m
  plowe-0: 3742-4, 6d.19h.38m
filterset (user): [array of 361 redacted]
trustedset: 
 added: [array of 41 redacted]
 removed: 
  about-scheme
switchRuleset: 
 added: [array of 2 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
 allReadyAfter: 115 ms (selfie)
 maxAssetCacheWait: 57 ms
popupPanel: 
 blocked: 10
 network: 
  amp.akamaized.net: 3
  analytics.edgekey.net: 1
  go-mpulse.net: 1
  googletagmanager.com: 2
  imasdk.googleapis.com: 1
  newsmaxfeednetwork.com: 2
```
</details>

**Comment by gorhill** [comment]
Unable to reproduce any issue, audio works fine after I unmute the video at the provided URL.

**Comment by stephenhawk8054** [comment]
Audio plays fine for me too.

**Comment by gorhill** [comment]
> when i turn off uBO and refresh page, audio still doesnt work on Firefox+newsmax

Closing as _invalid_, I doubt it's related to uBO.

**Comment by NewUser12345123** [comment]
Yeah like i said, when you disabled UBO and restart firefox, audio is fine, until you enable UBO

so clearly its UBO

also i didnt change anything and it was working the day before i posted this issue

How could it now be a UBO issue if when i turn off UBO it works, and when i turn on UBO it doesnt work

explain that guys????


clearly its UBO

**Comment by NewUser12345123** [comment]
Ok so i just uninstalled uBO and reinstalled with fresh version, no filters at all.
right when i turn it on, load the page, boom no sound.

so yeah its uBO

**Comment by stephenhawk8054** [comment]
As we cannot reproduce the issues (audio plays fine on my machine), we cannot investigate more.

**Comment by NewUser12345123** [comment]
I myself am having problems reproducing the issue as well on a virtual machine...
So i am trying to think outside the box and will troubleshoot more.
I will let you know if i learn anything pertaining to the issue. Thank you for your time.

**Comment by NewUser12345123** [comment]
Guys i figured it out! Im retarded. It was a conflict between uBO and "Audio Equalizer" addon (that is a recommended addon by Firefox.)

So i disabled the Audio Equalizer addon and Boom uBO immediately works as expected

SOOO yeah at this point just saying thank you for your guys' help trying to figure it out, really thanks for the time

and yeah just letting you know now that there is a conflict between the two addons
and apparently the issue is that.. if "Audio Equalizer" is enabled at all, while uBO is enabled, then there will be no audio
but it only does this (to my personal knowledge) on "NewsmaxTV.com" (m sure if its happening there its happening somewhere else)
when testing the Audio Equalizer app, it didnt matter what settings i used, if it was enabled at all with uBO then it just wouldnt produce sound.

This might be an issue for them, but i just want you guys to be aware, and yeah this just happened as it was working for the longest time and ive had both addons for years



thanks guys!!!! favorite addon and i tell like hundreds of people to get it
always install uBO on all my friends PCs and Phones lol



thanks so much and good luck!
